mitmproxy/addons/script.py

```diff
@@ -205,7 +205,10 @@ class ScriptLoader:
         An addon that manages loading scripts from options.
     """
     def run_once(self, command, flows):
-        sc = Script(command)
+        try:
+            sc = Script(command)
+        except ValueError as e:
```

Comment:
Why do you cast this to an OptionsError? Options are not involved here, we just should let the ValueError propagate and document that.